### PR TITLE
Add an extra ApeironNodes Seed

### PR DESCRIPTION
**Pull request by Panvasi**

### CODE REVIEW

File: chain4energy/chain.json [hunk 1]
```diff
@@ -177,6 +177,11 @@
   },
   "peers": {
     "seeds": [
+      {
```
Comment (JeremyParish69) [inline chain4energy/chain.json:180]: Please add to the bottom of the array

**Review by Panvasi — COMMENTED** [review]
Looking good👍

File: chain4energy/chain.json (hunk 1 above)
Comment (JeremyParish69) [reply chain4energy/chain.json:180]: This conversation was not resolved.

Comment (Panvasi) [reply chain4energy/chain.json]: > This conversation was not resolved.

Now?

Comment (JeremyParish69) [reply chain4energy/chain.json]: No, it's still not at the very bottom, but I see you've added it after an existing Seed on the same provider, and it's not worth additional back and forth.

**Pull request merged**